src/test/java/http_server/ResponseMakerTest.java

```diff
@@ -2,39 +2,93 @@
 
 import org.junit.Test;
 
+import static http_server.StatusCodes.*;
 import static org.junit.Assert.assertEquals;
 
 public class ResponseMakerTest {
 
+    private String responseStart = "HTTP/1.1 ";
+    private String space = " ";
+
+    private String buildResponse(String responseStart, String statusCode, String space, String statusMessage) {
```

Comment:
Is there code really similar to this in your CUT (class under test, i.e., ResponseMaker)? I much prefer my tests to be more explicit, so in this case I'd consider having the string your asserting against in each test to be hard-coded within that test instead of using a method to construct it. I prefer duplication (in tests) when it makes each test more understandable in isolation.